src/Symfony/Component/Security/Csrf/TokenGenerator/UriSafeTokenGenerator.php

```diff
@@ -38,7 +38,8 @@ public function generateToken()
         // Generate an URI safe base64 encoded string that does not contain "+",
         // "/" or "=" which need to be URL encoded and make URLs unnecessarily
         // longer.
-        $bytes = random_bytes($this->entropy / 8);
+        $length = max(1, abs((int) ($this->entropy / 8)));
```

Comment:
can entropy really be negative? I guess by definition it's non-negative. And is it correct then to use the absolute value? Or was your idea to make it positive when any code accidently adds a negative value? For now you would get an php error and now the code would run and use the absolute value, is this correct?
Or maybe validate in the constructor for a positive or non-negative value?